mb_collapse_work_attributes.user.js

```diff
@@ -9,8 +9,8 @@
 // @updateURL    https://raw.github.com/ROpdebee/mb-userscripts/main/mb_collapse_work_attributes.user.js
 // @match        *://*.musicbrainz.org/work/*
 // @match        *://musicbrainz.org/work/*
-// @match        *://*.musicbrainz.org/artist/*/works
-// @match        *://musicbrainz.org/artist/*/works
+// @match        *://*.musicbrainz.org/artist/*/works*
+// @match        *://musicbrainz.org/artist/*/works*
```

Comment:
```suggestion
// @match        *://*.musicbrainz.org/artist/*/works
// @match        *://*.musicbrainz.org/artist/*/works?*
```
This might be the safer way to do this with match patterns but I still have to test this with VM and TM.

Reply:
Works on my end, will update and force-push. Also going to simplify the other `@match` URLs to use *.mb instead of duplicating.

Btw, if you're looking for an artist to test on, Dolly Parton is one.

Reply:
Yes, I also tested this with Dolly Parton right now ;)